test/unittests/execute_test.cpp

```diff
@@ -1441,3 +1441,24 @@ TEST(execute, start_section)
     EXPECT_EQ(ret[0], 42);
     EXPECT_EQ(instance.memory.substr(0, 4), from_hex("2a000000"));
 }
+
+TEST(execute, data_section)
```

Comment:
Probably better to move the test to instantiate_tests.cpp (and then it doesn't need `execute`)

Reply:
I can do that. @chfast any comments?

Reply:
Yes, it makes sense to move it.

Reply:
Done.